docs/BuildConfiguration.md

```diff
@@ -125,8 +125,9 @@ different locations around the local filesystem.
 
 As a general rule, references to data files are specified as absolute URIs and must start with the protocol name.   
 Example:   
-Local files: `"file:///Users/kelvin/otp/streetGraph.obj"`   
-Google Cloud Storage files: `"gs://otp-test-bucket/a/b/graph.obj"`
+Local files: `"file:///Users/kelvin/otp/streetGraph.obj"`  
+HTTPS resources: `"https://download.geofabrik.de/europe/norway-latest.osm.pbf"`  
+Google Cloud Storage files: `"gs://otp-test-bucket/a/b/graph.obj"`  
```

Comment:
Should we add a mention of `osmCacheDataInMem` somewhere, or even enable it by default, if it is a https source?

Reply:
We'll do this later